drake/solvers/MobyLCP.cpp

```diff
@@ -129,20 +129,20 @@ bool MobyLCPSolver::solve(OptimizationProblem& prog) const {
   // Restriction 3 could reasonably be relaxed to simply let unbound
   // variables sit at 0.
 
-  assert(prog.getGenericConstraints().empty());
-  assert(prog.getGenericObjectives().empty());
-  assert(prog.getAllLinearConstraints().empty());
-  assert(prog.getBoundingBoxConstraints().empty());
+  assert(prog.get_generic_constraints().empty());
+  assert(prog.get_generic_objectives().empty());
+  assert(prog.get_all_linearConstraints().empty());
```

Comment:
Since this is passing CI, I guess it must be OK, but I don't understand why it builds, since this token doesn't appear anywhere else.